bindings/python/src/pgvecto_rs/highapi/record.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional, Type, Union
+from uuid import UUID, uuid4
+
+from numpy import array, float32, ndarray
+from sqlalchemy.orm import DeclarativeBase, Mapped
+
+
+class RecordORM(DeclarativeBase):
+    __tablename__: str
+    id: Mapped[UUID]
+    text: Mapped[str]
+    meta: Mapped[dict]
+    embedding: Mapped[ndarray]
```

Comment:
Why is it a numpy.ndarray instead of List[float]?

Reply:
The underlying driver is designed to use np.ndarry first (following pgvector)